src/commands/up.rs

```diff
@@ -35,6 +36,9 @@ const MULTI_TRIGGER_START_OFFSET: tokio::time::Duration = tokio::time::Duration:
 // any exited" check.
 const MULTI_TRIGGER_LET_ALL_START: tokio::time::Duration = tokio::time::Duration::from_millis(500);
 
+// Map of component ID to trigger ID
+type ComponentTriggerMap = HashMap<String, String>;
```

Comment:
This really should be e.g. `HashMap<String, HashSet<String>>`; one component can have multiple triggers pointing at it.

Reply:
Good catch. Thank you!